mongodbatlas/resource_mongodbatlas_cloud_provider_access_setup_test.go

```diff
@@ -41,40 +39,6 @@ func TestAccConfigRSCloudProviderAccessSetupAWS_basic(t *testing.T) {
 	)
 }
 
-func TestAccConfigRSCloudProviderAccessSetupAWS_importBasic(t *testing.T) {
```

Comment:
question: why was this test deleted?

Reply:
good catch. This acceptance test is testing the import function so it should still be relevant